app/views/partials/_navbar.html.erb

```diff
@@ -1,16 +1,11 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light" id="navbar">
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsing-navbar">
-    <span class="navbar-toggler-icon"></span>
-  </button>
-  <div class="navbar-collapse collapse" id="collapsing-navbar">
-    <ul class="navbar-nav">
+<nav class="navbar navbar-light" style="background-color: #e3f2fd;" id="navbar">
+  <div class= id="navbarText">
+    <ul class="nav">
       <li class="nav-item active">
-        <%= link_to ' Home', root_path, class: 'glyphicon glyphicon-home' %>
+        <a class="nav-link" href="/restaurant/index">Home</a>
       </li>
-    </ul>
-    <ul class="navbar-nav pull-right">
```

Comment:
You removed the pull-right here but it's still in the .scss. You can try to integrate in line 7 next to nav-item or if you don't want to use it then remove it from the .scss